xml/cl.xml

```diff
@@ -1177,7 +1177,8 @@ server's OpenCL/api-docs repository.
         <enum value="0x1067"        name="CL_DEVICE_PREFERRED_WORK_GROUP_SIZE_MULTIPLE"/>
         <enum value="0x1068"        name="CL_DEVICE_WORK_GROUP_COLLECTIVE_FUNCTIONS_SUPPORT"/>
         <enum value="0x1069"        name="CL_DEVICE_GENERIC_ADDRESS_SPACE_SUPPORT"/>
-            <unused start="0x106A" end="0x106E" comment="Reserved for upcoming KHR extension"/>
+        <enum value="0x106A"        name="CL_DEVICE_LATEST_CTS_VERSION_TESTED"/>
```

Comment:
I think we should use enum `0x1072` for this query, since enum `0x106A` is already assigned for an upcoming KHR extension.

Reply:
Well, Github should really be the place for reserving enums. It's not reasonable to expect people to look at all open MRs/branches in Gitlab and scan for assigned enums IMHO.

Reply:
So, this actually was a public extension for which we hadn't reserved enums yet (I've opened https://github.com/KhronosGroup/OpenCL-Docs/pull/342). I'll change the enum.